pages/levels/3.vue

```diff
@@ -0,0 +1,174 @@
+<script lang="ts">
+import { defineComponent, ref } from "vue";
+import { VueDraggable } from "vue-draggable-plus";
+
+const code = `    <p><span class="axolotlText"></span></p>`;
+const pool = ref([
+  {
+    name: "About | accessolotl",
+    id: "h1",
+  },
+  {
+    name: "Our Team",
+    id: "t1",
+  },
+  {
+    name: "Top 10 Axolotl Facts - accessolotl",
+    id: "h2",
+  },
+  {
+    name: "How does regeneration work?",
+    id: "t2",
+  },
+  {
+    name: "How are they different from salamanders?",
+    id: "t3",
+  },
+  {
+    name: "accessolotl - homepage - imprint",
+    id: "h3",
+  },
+  {
+    name: "Why axolotl is simply the coolest animal",
+    id: "t4",
+  },
+]);
+const titles = ref([]);
+const headings = ref([]);
+
+export default defineComponent({
+  data: () => {
+    return {
+      markdown: `Title or heading, what should I use?
+
+In web development, there is an important difference between headings and titles. A title is meant to inform users about the current location of the page, without them requiring to read the page content. On the other hand, headings help users find specific content within the page.
+
+You could say, user agents (e.g. Browsers like Chrome or Firefox) display titles as the name of the tab containing the page, while browsers often allow to navigate from heading to heading within a page.
+
+Titles also help with SEO, if used correctly. Here's a little checklist what your title should look like:
+
+- Include Keyword
+- Use Keyword At the Beginning of page title
+- Click-worthy title
+- Use Business Name
+- Optimize Length
+
+## Links
+- [Providing titles for Web pages](https://www.w3.org/WAI/WCAG22/Techniques/general/G88)
+- [Providing headings](https://www.w3.org/WAI/WCAG22/Techniques/general/G130.html)
+- [Writing better titles](https://contentcreation.ai/writing-better-titles/)`,
+
+      code,
+      defaultCode: code,
+      task: "Is it a title or a heading?",
+      pool,
+      titles,
+      headings,
+    };
+  },
+  computed: {
+    checkTitles(): boolean {
+      if (this.titles.length === 0) return false;
```

Comment:
Sinnloser Check weil 0 !== 4, also das wird ja auch in Zeile 74 überprüft
gleiches in Zeile 78